### DISCUSSION TITLE
Is it possible to make a wiki and issues public while the code is private?

### DISCUSSION THREAD

**Discussion by velara3** [community/community#44255] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Hi, 

I have a private repository and I would like to know if it's possible to make the wiki and the issues public? I see there is a similar question on making the **issues** public located here: 

https://github.com/orgs/community/discussions/4264

From the discussion it looks like you have to ask to have it enabled? Is this correct? Any update on if it will be an option for everyone and for wiki pages? 

And, I also see that it is a **_custom issues form_** that you create rather than the standard issues tracker. This might be a separate question then because I'm ok with my users using the standard issues form so that it integrates into the project. I don't know if custom issues forms do that. So this might be a separate request. 

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

**Comment by ghost** [comment]
hi velara you can use issues in private repos but not discussions

**Reply by ghost** [reply]
sorry not disscussions it wiki
